docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -53,6 +53,19 @@ yarn add @material-ui/core@next
   +<BottomNavigation onChange={(event: React.SyntheticEvent) => {}} />
   ```
 
+### Button
+
+- The button `color` prop is now "primary" by default, and "default" has been deprecated.
+
+  ```diff
+  <Button color="primary">Primary</Button>
+  <Button>Primary</Button>
+  ```
+
+  ```
+  /(<Button(\n*.*?)*?)color="primary"/$1/g
+  ```
```

Comment:
What does it mean?

Reply:
It's a regex (in Vim search and replace format) that removes color="primary" from Buttons.

Reply:
Ok, maybe we should explain? I don't know how to use it. Do we have something that can work with VScode?

Reply:
Let's omit it.